.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.0
```

Comment:
I would rather not include this change, its specific to the developer environment.